site2/docs/client-libraries-cpp.md

```diff
@@ -10,7 +10,7 @@ All the methods in producer, consumer, and reader of a C++ client are thread-saf
 
 ## Supported platforms
 
-Pulsar C++ client is supported on **Linux** and **MacOS** platforms.
+Pulsar C++ client is supported on **Linux** ,**MacOS** and **Windows(64-bit)** platforms.
```

Comment:
Windows 32 bit is supported after https://github.com/apache/pulsar/pull/11082 (from 2.8.1).

Reply:
@BewareMyPower Thanks for your comment.
@fengtao1998 you can update the Pulsar 2.8.1 doc with the above comment as well. Thank you.